src/styles/components/_definition.scss

```diff
@@ -1,35 +1,71 @@
 $breakpoint-tablet: 820px;
 
 .cmp-definition {
-  min-height: 20rem;
+  position: relative;
+  line-height: 1.75rem;
 
   &__heading {
     position: relative;
     font-family: Viga, sans-serif;
     font-weight: 400;
-    font-size: map.get($font-sizes, "md");
+    font-size: 1.5625rem;
+
+    /* Circle around "definition" */
+    &::after {
+      content: "";
+      position: absolute;
+      top: -1.8rem;
+      left: -1.8rem;
+      width: 15rem;
+      height: 6rem;
+      background-size: 80%;
+      background-image: url("../../../public/images/shapes/circle.svg");
+      background-repeat: no-repeat;
+    }
+  }
+
+  &__text {
+    padding: 4rem 0;
+    max-width: 60ch;
   }
 
-  &__heading-circle {
-    position: absolute;
-    width: 14rem;
-    height: auto;
-    top: -1.8rem;
-    left: -2rem;
+  &__content {
+    font-size: map.get($font-sizes, "xs");
+    letter-spacing: 0.02rem;
   }
 
   @media only screen and (min-width: $breakpoint-tablet) {
-    background:
-      linear-gradient(to right, white 50%, rgba(255 255 255 / 0%)),
-      url("../../../public/images/definition_text_light.svg");
-    background-size: contain;
-    background-position-x: right;
-    background-repeat: no-repeat;
-    margin: auto;
-    max-width: 59rem;
+    &__bg-text {
+      position: absolute;
+      top: -2rem;
+      right: -8rem;
+      width: 100%;
+      height: 110%;
+      background:
+        linear-gradient(to right, white 40ch, rgba(255 255 255 / 0%)),
+        url("../../../public/images/definition_text_light.svg");
+      background-size: contain;
+      background-position: 100%;
+      background-repeat: repeat-x;
+      margin: auto;
+      z-index: -1;
+    }
```

Comment:
I was just looking at the dark mode version for this. If we want the `Definition` image text to go off screen like the design, we could do something like: `background-position: center right -15px;` (in this screenshot, the left-hand side where all the black is my browser)
<img width="603" alt="Screen Shot 2022-11-09 at 7 14 42 AM" src="https://user-images.githubusercontent.com/69602589/200827909-0054cc6c-e8d9-4a04-b0fe-b6d7cf25a248.png">